### PR TITLE
Update top-level-templates.md

### LINKED ISSUE #26313: Please document how to change default console template.
Could you please document how we can change the console template back to the 'classic' Program + Main.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 447e23ae-0256-d8c1-4dca-d31fe21602a5
* Version Independent ID: 926db74c-be79-f7cd-a4e9-e156dafa6287
* Content: [C# template changes in .NET 6 - .NET](https://docs.microsoft.com/en-us/dotnet/core/tutorials/top-level-templates)
* Content Source: [docs/core/tutorials/top-level-templates.md](https://github.com/dotnet/docs/blob/main/docs/core/tutorials/top-level-templates.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @tdykstra
* Microsoft Alias: **tdykstra**

### PR DESCRIPTION
**Pull request by adegeo**
## Summary

- Describe how to override the new template style

Fixes #26313

### CODE REVIEW

**Comment by adegeo** [comment]
@BillWagner

**Review by tdykstra — COMMENTED** [review]
Good start, but we should also tell people how to do it in VS -- create .NET 5 project and change .csproj to target .net 6.

Also to elicit feedback, the new section could link to this issue and ask people to add their comments/votes.

File: docs/core/tutorials/top-level-templates.md [hunk 1]
```diff
@@ -77,3 +79,11 @@ You can also add a [`<Using>`](../project-sdk/msbuild-props.md#using) item in yo
   <Using Remove="System.Net.Http" />
 </ItemGroup>
 ```
+
+## Use the old program style
```
Comment (BillWagner) [inline docs/core/tutorials/top-level-templates.md:83]: I agree with @tdykstra 's general comments to include in this section. Then, this is ready to :shipit:

**Review by BillWagner — APPROVED** [review]
Thanks @adegeo 

After you add the comments from Tom, this is ready to :shipit:

**Comment by Walluce** [comment]
I would prefer a setting in Visual Studio that lets me choose between old and new templates, by default.

**Comment by jackfagner** [comment]
> Good start, but we should also tell people how to do it in VS -- create .NET 5 project and change .csproj to target .net 6.
> 
> Also to elicit feedback, the new section could link to this issue and ask people to add their comments/votes.

Before you can actually create a .NET 5 project you need to download and install the .NET 5 SDK, which is not included in VS 2022. It's a lot of work just to get a code template that allow you to change the return type of "main" or add another method to the Program class.

**Pull request merged**